Powershell-PoC-Environment-Setup.ps1

```diff
@@ -242,6 +432,8 @@ $rgStorage = New-AzureRmResourceGroup -Name "poc-storage-rg" -Location $eastLoca
 
 # Subnet Creation
 Write-Verbose "Creating Subnets..." -Verbose

```

Comment:
Well the write with time only prints user defined strings to the console. The Write-Verbose prints more transactional type information that the Write-WithTime doesn't include, so they're not really duplicates after-all, but complement each other.